dwave/cloud/computation.py

```diff
@@ -1,5 +1,15 @@
 """
-TODO: short description about computation
+Computation manages the interactions between your code and a :term:`solver`, which
+manages interactions between the remote resource and your submitted problems.
+
+Your solver instantiates a :class:`Future` object for its calls, via D-Wave Sampler API
+(SAPI) servers, to the remote resource.
+
+You can interact with the :class:`Future` object for pending calls: monitoring problem status,
+waiting for and retrieving results, cancelling enqueued jobs, etc.
+
+Some :class:`Future` methods are blocking.
+
 """
```

Comment:
I would also clarify the meaning behind "computation" name. Through computation module we prove to users an interface to the running (or completed) computation (sampling on QPU or sw) that's happening on the remote side. `Future` object allows them to query the current state of that computation and read the result when available.

Reply:
Done